src/Microsoft.Health.Dicom.Core/Exceptions/DicomElementValidationException.cs

```diff
@@ -10,25 +10,35 @@ namespace Microsoft.Health.Dicom.Core.Exceptions
 {
     public class DicomElementValidationException : ValidationException
     {
-        public DicomElementValidationException(string name, string value, DicomVR vr, string message)
+        public DicomElementValidationException(string name, DicomVR vr, string message)
+            : base(message)
+        {
+            EnsureArg.IsNotNull(name, nameof(name));
+            EnsureArg.IsNotNull(vr, nameof(vr));
+            Name = name;
+            VR = vr;
+        }
+
+        public DicomElementValidationException(string name, DicomVR vr, string message, string value)
            : base(message)
         {
             EnsureArg.IsNotNull(name, nameof(name));
-            EnsureArg.IsNotNull(value, nameof(value));
             EnsureArg.IsNotNull(vr, nameof(vr));
-            EnsureArg.IsNotNull(message, nameof(message));
+            EnsureArg.IsNotNull(value, nameof(value));
 
             Name = name;
-            Value = value;
             VR = vr;
+            Value = value;
         }
 
-        public override string Message => string.Format(DicomCoreResource.DicomElementValidationFailed, Name, Value, VR.Code, base.Message);
+        public override string Message => Value == null
```

Comment:
I think we should check the message too. If the user passed their own message, I don't think we need to preface it with another one.

Reply:
This is existing behavior.  I think the idea is to given a general error message, along with detailed information. I'm not sure if should change it or not.